packages/perspective-viewer-d3fc/src/js/legend/scrollableLegend.js

```diff
@@ -26,7 +28,7 @@ export default fromLegend => {
             pageIndex = 0;
             domain = newDomain;
         }
-        pageCount = Math.ceil(domain.length / pageSize);
+        pageCount = calculatePageCount();

```

Comment:
If you call `calculatePageCount(pageSize)` (i.e. pass the parameter), will it simplify the implementation of `calculatePageCount`?

Reply:
Good call.